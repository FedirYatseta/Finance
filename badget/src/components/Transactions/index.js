import React from 'react'
import Transaction from '../Transaction/index'


const Transactions = ({ transactions = [] }) => {
    debugger
    return (
        <div>
            {transactions.map((transaction) => <Transaction key={transaction.id} transaction={transaction} />)}
            <hr />
        </div>
    )
}

export default Transactions
