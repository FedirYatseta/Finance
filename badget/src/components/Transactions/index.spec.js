

//import { shallow } from 'enzyme';
import React from 'react';
import renderer from 'react-test-renderer';
import Transactions from '.';



describe('Transactions', () => {
  it('should show Transactions', () => {
    const transactions = []
    const component = renderer.create(
      <Transactions transactions={transactions} />,
    );
    expect(component).toMatchSnapshot();

  })
});