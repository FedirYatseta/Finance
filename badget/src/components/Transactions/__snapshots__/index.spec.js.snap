// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Transactions should show Transactions 1`] = `
<div>
  <hr />
</div>
`;
